public/stylesheets/landing.css

```diff
@@ -2,13 +2,21 @@ body {
   background-color: #000;
 }
 
+h1 {
```

Comment:
It's usually better to give HTML element some CSS class and style it the class. But not an issue here as this stylesheet is used only in one page of an app.